torch/csrc/jit/python/init.cpp

```diff
@@ -292,6 +293,37 @@ void initJITBindings(PyObject* module) {
       .def("_jit_set_autocast_mode", &setAutocastMode)
       .def("_jit_pass_onnx_lint", ONNXLintGraph)
       .def("_jit_pass_onnx_function_extraction", onnx::ONNXFunctionExtraction)
+      .def(
+          "_jit_onnx_logging_enabled",
+          []() { return ::torch::jit::get_onnx_logging_enabled(); })
+      .def(
+          "_jit_set_onnx_logging_enabled",
+          [](bool enabled) -> void {
+            ::torch::jit::set_onnx_logging_enabled(enabled);
+          })
+      .def(
+          "_jit_set_onnx_logging_stream",
+          [](std::string stream_name) -> void {
+            if (stream_name == "stdout") {
+              ::torch::jit::set_onnx_logging_output_stream(std::cout);
+            } else if (stream_name == "stderr") {
+              ::torch::jit::set_onnx_logging_output_stream(std::cerr);
+            } else {
+              std::cerr << "ERROR: only `stdout` and `stderr`"
+                        << "are supported as output options" << std::endl;
+            }
+          })
```

Comment:
Have you considered merging this code into `jit_init`? There can be a default value (disabled), but the user could enable it and pass the desired stream during jit initialization.

This way there is no need to add more APIs

Reply:
merged the two